src/Mizar/MizarMainWindow.ui

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>MainWindow</class>
+ <widget class="QMainWindow" name="MainWindow">
```

Comment:
Please be consistent with the "name" property. Either use `snake_case_`, `camelCase`, or `PascalCase`.

Reply:
Same for the other ui file.

Reply:
Discussed offline. Agreed to keep the widget names PascalCase and make everything else snake_case_